bin/commands/scripts/build_watch.js

```diff
@@ -1,3 +1,3 @@
 const build = require('./build');
 
-module.exports = options => build({ ...options, flags: ['--watch', ...options.flags] });
+module.exports = options => build({ ...options, flags: ['-w', ...options.flags] });
```

Comment:
Does `--watch` not work? Not a big deal, but I kind of like using the longhand as it is more descriptive -- no need to look up what the shorthand means if you aren't familiar.

Reply:
I figured since the file name is `build_watch`, the flag meant a watch argument. I can change it to `--watch` tho.